.eslintrc

```diff
@@ -22,5 +22,6 @@
       "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
       "react/forbid-prop-types": "off",
       "react/no-array-index-key": "off",
+      "linebreak-style": "off",
```

Comment:
Could you use this .gitattributes specific rule instead?:

`*.js text eol=lf`

(source: http://eslint.org/docs/rules/linebreak-style)